sample_integration/sample_integration/AppDelegate.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  AppDelegate.swift
+//  sample_integration
+//
+//  Created by Eric Lanz on 6/4/20.
+//  Copyright © 2020 Embrace. All rights reserved.
+//
+
+import UIKit
+// EMBRACE HINT:
+// Embrace is integrated using cocoapods.  If you just downloaded the project and it is failing to build, simply run
+// 'pod update' from the root of the project in terminal,  See: https://docs.embrace.io/docs/ios-integration-guide
+import Embrace
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        
+        // EMBRACE HINT:
+        // Embrace can use either objective-c or swift style logging, to enable swift style logging set this flag to true
+        // This can be done before calling start to ensure all logging is in the the clean format
+        Embrace.sharedInstance()?.setCleanLogsEnabled(true)
+        
+        // EMBRACE HINT:
+        // Always initialize Embrace as early as possible and in-line with the launch methods your application is using
```

Comment:
i'd add something about if they are using firebase/crashlytics, we should be initialized before that to capture crashes reliably